research/lamp-automated-model-parallelism/README.md

```diff
@@ -17,21 +17,28 @@ MICCAI 2020 (Early Accept, paper link: https://arxiv.org/abs/2006.12575)
 ## To run the demo:
 
 ### Prerequisites
-- install the latest version of MONAI: `git clone https://github.com/Project-MONAI/MONAI` and `pip install -e .`
+- Download and switch to MONAI 0.2.0 source code:
+```bash
+git clone https://github.com/Project-MONAI/MONAI
+cd MONAI
+git checkout 0.2.0
+pip install -e .  # install from the source code
```

Comment:
Similar to the coplenet, maybe simplify to below?
```
pip install -e '.[torchgpipe]`
```

Thanks.

Reply:
why do we add `torchgpipe`? there're packages like matplotlib, Jupiter notebook in the examples, we don't introduce them as (optional) deps

Reply:
Oh, I see, so this `torchgpipe` is not a common lib also for other works, right?